Testing.Rmd

```diff
@@ -596,5 +1364,101 @@ There are a few things to bear in mind:
   32-bit versions of R) so use `expect_equal()` rather than
   `expect_identical()`.
 
-[tdd]:https://en.wikipedia.org/wiki/Test-driven_development
-[extreme-programming]:https://en.wikipedia.org/wiki/Extreme_programming
+CRAN revision thought dump:
+
+* Enrich description of platforms, i.e. say more than "Windows, Mac, Linux".
+  The check flavors: <https://cran.r-project.org/web/checks/check_flavors.html>.
+  - Draw a line to how this determines which "flavors" a package should be
+    checked on.
+    Depends on how broad the user base is and the "trickiness" of the package's
+    code (e.g. does it need compilation?).
+    Do a cost benefit analysis.
+    Connect that to which `usethis::use_github_action_check_*()` variant is
+    appropriate.
+* How to decide whether to let a test run on CRAN.
```

Comment:
Any test that depends on something that is not under your direct control is a candidate for skipping on CRAN.